Packs/FeedFeedly/Integrations/FeedFeedly/FeedFeedly.py

```diff
@@ -0,0 +1,994 @@
+    "intrusion-set": ThreatIntel.ObjectsNames.INTRUSION_SET,
+}
+
+MITRE_CHAIN_PHASES_TO_DEMISTO_FIELDS = {
+    "build-capabilities": ThreatIntel.KillChainPhases.BUILD_CAPABILITIES,
+    "privilege-escalation": ThreatIntel.KillChainPhases.PRIVILEGE_ESCALATION,
+    "adversary-opsec": ThreatIntel.KillChainPhases.ADVERSARY_OPSEC,
+    "credential-access": ThreatIntel.KillChainPhases.CREDENTIAL_ACCESS,
+    "exfiltration": ThreatIntel.KillChainPhases.EXFILTRATION,
+    "lateral-movement": ThreatIntel.KillChainPhases.LATERAL_MOVEMENT,
+    "defense-evasion": ThreatIntel.KillChainPhases.DEFENSE_EVASION,
+    "persistence": ThreatIntel.KillChainPhases.PERSISTENCE,
+    "collection": ThreatIntel.KillChainPhases.COLLECTION,
+    "impact": ThreatIntel.KillChainPhases.IMPACT,
+    "initial-access": ThreatIntel.KillChainPhases.INITIAL_ACCESS,
+    "discovery": ThreatIntel.KillChainPhases.DISCOVERY,
+    "execution": ThreatIntel.KillChainPhases.EXECUTION,
+    "installation": ThreatIntel.KillChainPhases.INSTALLATION,
+    "delivery": ThreatIntel.KillChainPhases.DELIVERY,
+    "weaponization": ThreatIntel.KillChainPhases.WEAPONIZATION,
+    "act-on-objectives": ThreatIntel.KillChainPhases.ACT_ON_OBJECTIVES,
+    "command-and-control": ThreatIntel.KillChainPhases.COMMAND_AND_CONTROL,
+}
+
+STIX_2_TYPES_TO_CORTEX_CIDR_TYPES = {
+    "ipv4-addr": FeedIndicatorType.CIDR,
+    "ipv6-addr": FeedIndicatorType.IPv6CIDR,
+}
+
+THREAT_INTEL_TYPE_TO_DEMISTO_TYPES = {
+    "campaign": ThreatIntel.ObjectsNames.CAMPAIGN,
+    "attack-pattern": ThreatIntel.ObjectsNames.ATTACK_PATTERN,
+    "report": ThreatIntel.ObjectsNames.REPORT,
+    "malware": ThreatIntel.ObjectsNames.MALWARE,
+    "course-of-action": ThreatIntel.ObjectsNames.COURSE_OF_ACTION,
+    "intrusion-set": ThreatIntel.ObjectsNames.INTRUSION_SET,
+    "tool": ThreatIntel.ObjectsNames.TOOL,
+    "threat-actor": ThreatIntel.ObjectsNames.THREAT_ACTOR,
+    "infrastructure": ThreatIntel.ObjectsNames.INFRASTRUCTURE,
+}
+
+
+class Client(BaseClient):
+    def fetch_indicators_from_stream(self, stream_id: str, newer_than: float, *, limit: Optional[int] = None) -> list:
+        params = {
+            "streamId": stream_id,
+            "count": 20,
+            "newerThan": int(newer_than * 1_000),
+            "client": "feedly.demisto.client",
+        }
+
+        objects = []
+
+        while True:
+            resp = self._http_request("GET", "/v3/enterprise/ioc", params=params, resp_type="response")
+            objects.extend(resp.json().get("objects", []))
+
+            if "link" not in resp.headers:
+                break
+
+            next_url = resp.headers["link"][1:].split(">")[0]
+            params["continuation"] = parse_qs(next_url)["continuation"][0]
+
+        demisto.debug(f"Fetched {len(objects)} objects from stream {stream_id}")
+
+        indicators = STIX2Parser().parse_stix2_objects(objects)
+
+        if limit:
+            indicators = indicators[:limit]
```

Comment:
@MichaelYochpaz the issue with the limit was that I was applying it on the stix `objects` instead of the XSOAR `indicators`

I think 2 of my intended commits got merged because of unsuccesful pre-commit validation, which is why the fix doesn't appear in my commit messages :)